ui-tests/src/test/java/org/openmrs/reference/AddFormTest.java

```diff
@@ -18,22 +16,36 @@
  */
 public class AddFormTest extends TestBase {
     private HomePage homePage;
+    private static String name = "newFormTest";
+    private static String description = "description of new form";
+    private static  String version = "1.2";
     private HeaderPage headerPage;
     private ManageFormsPage manageForm;
     private ClinicianFacingPatientDashboardPage patientDashboardPage;
+    private AdministrationPage administrationPage;
+    private ManageHtmlFormsPage manageHtmlFormsPage;
+    private HtmlFormsPage htmlFormsPage;
 
     @Before
     public void setUp() throws Exception {
         homePage = new HomePage(page);
-        assertPage(homePage);
         headerPage = new HeaderPage(driver);
+        administrationPage = new AdministrationPage(page);
         manageForm = new ManageFormsPage(driver);
         patientDashboardPage = new ClinicianFacingPatientDashboardPage(page);
+
     }
 
-    @Ignore //ignore due to moving forms functionality
     @Test
```

Comment:
you need @Category(BuildTests.class) on each Test

Reply:
Resolved thanks